src/_langs/es/_betterbook.yaml

```diff
@@ -1,5 +1,5 @@
 toc:
-  - title: "Web <strong>Fundamentals</strong>"
+  - title: "Aspectos básicos de la <strong>Web</strong>"
```

Comment:
Is there a better word here?  It is not really the basics, it is the core principles.